lib/ConditionBuilder.js

```diff
@@ -1,69 +1,89 @@
 var assert = require('assert')
 var typ = require('typ')
 var typeUtil = require('./typeUtil')
-var ConditionJunction = require('./ConditionJunction')
+var util = require('util')
 
 /**
- * @param {Object=} opt_conditions
+ * @param {Object=} opt_exprs
```

Comment:
ya, agreed. improved the docs in a couple places. also i just removed this argument, because i think it's probably more confusing than useful.